src/dxgi/dxgi_output.cpp

```diff
@@ -300,28 +294,27 @@ namespace dxvk {
 
     // Walk over all modes that the display supports and
     // return those that match the requested format etc.
-    DEVMODEW devMode = { };
-    devMode.dmSize = sizeof(DEVMODEW);
+    wsi::WsiMode devMode = { };
     
     uint32_t srcModeId = 0;
     uint32_t dstModeId = 0;
     
     std::vector<DXGI_MODE_DESC1> modeList;
     
-    while (GetMonitorDisplayMode(m_monitor, srcModeId++, &devMode)) {
+    while (wsi::getDisplayMode(m_monitor, srcModeId++, &devMode)) {
       // Skip interlaced modes altogether
-      if (devMode.dmDisplayFlags & DM_INTERLACED)
+      if (devMode.interlaced)
         continue;
       
       // Skip modes with incompatible formats
-      if (devMode.dmBitsPerPel != GetMonitorFormatBpp(EnumFormat))
+      if (devMode.bitsPerPixel != GetMonitorFormatBpp(EnumFormat))
         continue;
       
       if (pDesc != nullptr) {
         DXGI_MODE_DESC1 mode;
-        mode.Width            = devMode.dmPelsWidth;
-        mode.Height           = devMode.dmPelsHeight;
-        mode.RefreshRate      = { devMode.dmDisplayFrequency * 1000, 1000 };
+        mode.Width            = devMode.width;
```

Comment:
Any reason not to use `ConvertDisplayMode` here?
Can fix up the format afterwards.